app/lib/pre_assembly/from_staging_location/file_set.rb

```diff
@@ -53,6 +53,8 @@ def resource_type_descriptions
           resource_has_non_images ? 'object' : 'page'
         when :document
           'document'
+        when :geo
+          resource_has_non_images ? 'object' : 'preview'
         when :'3d'
```

Comment:
based on this example object: https://argo-stage.stanford.edu/view/druid:cj778hn3376